helm-chart/templates/ingress.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-{{ .Values.deployEnv }}
+  annotations:
+    {{- if eq .Values.sandbox "true" }}
+    kubernetes.io/ingress.class: nginx-ingress
+    {{- else }}
+    kubernetes.io/ingress.global-static-ip-name: "sefaria-{{ .Values.deployEnv }}"
+    {{- end }}
+spec:
+  tls:
+    - secretName: {{ template "secrets.originTls" . }}
+    - secretName: {{ template "secrets.originIlTls" . }}
```

Comment:
This section also switches on sandbox - all sandboxes use the wildcard cert.  Unless you are managing that the values file?  The IlTls reference is also only required in prod.

Reply:
Ya... missed that part. 

Will make the second secret to be conditional but I am handling the secret name from the Values file so we can let the first secret name be pulled from the values file. I've left a comment in the values file for the same.